frontend/src/views/teamInfo.js

```diff
@@ -52,17 +46,16 @@ class WinsCard extends Component {
   }
 
   render() {
-    const { wins, draws, losses } = this.state;
     return (
       <div className="card">
         <div className="content">
           <div className="col-2-row">
             <div className="row-items-box items-box-center items-box-skinny">
-              <label>wins</label>
+              <label>Wins:</label>
               <h1>{this.props.wins}</h1>
             </div>
             <div className="row-items-box items-box-center items-box-skinny">
-              <label>losses</label>
+              <label>Losses:</label>
```

Comment:
Drop the colons? More consistent with rest of page
(or if you want, you can put colons everywhere. higher effort tho)